apps/wallet-dashboard/components/SupplyIncreaseVestingOverview.tsx

```diff
@@ -81,7 +74,7 @@ export function SupplyIncreaseVestingOverview() {
             });
     }
 
-    return showSupplyIncreaseVestingOverview ? (
+    return !isSupplyIncreaseVestingScheduleEmpty ? (
```

Comment:
we are missing the staked condition
```suggestion
    return !isSupplyIncreaseVestingScheduleEmpty || supplyIncreaseVestingStakedMapped.length > 0 ? (
```